### PR TITLE
Interactive tree view

### PR DESCRIPTION
**Pull request by M6AI**
## Bug / Requirement Description
Interactive mode is not supported in UI tree view.

## Solution description
For now, the code is duplicated and appropriately updated from a combination of the static tree view and the interactive entries.

## Checklist:
- [x] Test
- [ ] Example (both test_plan.py and .rst)
- [ ] Documentation (API)
- [x] News fragment present for release notes
- [ ] MS info leakage check
- [ ] For new driver: driver index page
- [ ] For new assertion: ui/pdf/std renderers, documentation
- [ ] For new cmdline arg: documentation

### CODE REVIEW

File: testplan/web_ui/testing/src/Nav/InteractiveNav.js [hunk 1]
```diff
@@ -35,27 +35,57 @@ const InteractiveNav = (props) => {
...
-        filter={null}
-        displayEmpty={true}
-        displayTags={false}
-        displayTime={false}
-        selectedUid={GetSelectedUid(props.selected)}
-        handleClick={props.handleClick}
-        envCtrlCallback={props.envCtrlCallback}
-        url={props.url}
-      />
+      {renderNavigation(props)}
```
Comment (M6AI) [inline testplan/web_ui/testing/src/Nav/InteractiveNav.js:38]: Change here is just preparation for unifying the two `Nav`.

File: testplan/web_ui/testing/src/Nav/InteractiveTreeView.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
+import React from 'react';
```
Comment (M6AI) [inline testplan/web_ui/testing/src/Nav/InteractiveTreeView.js:1]: This is essentially duplication of the TreeView, but entries are interactive entries inside each node.

File: testplan/web_ui/testing/src/Nav/NavList.js [hunk 3]
```diff
@@ -1,67 +1,115 @@
...
-import {STATUS} from "../Common/defaults";
+import {
+    STATUS,
+    RUNTIME_STATUS,
+    NAV_ENTRY_ACTIONS,
+} from "../Common/defaults";
 
 /**
  * Render a vertical list of all the currently selected entries children.
+  * Render a vertical list of all the currently selected entries children for
```
Comment (zhenyu-ms) [inline testplan/web_ui/testing/src/Nav/NavList.js:17]: Looks like sort of duplicate with the line above.

Comment (M6AI) [reply testplan/web_ui/testing/src/Nav/NavList.js:17]: Indeed. Fixing, thanks @zhenyu-ms for catching it.

**Review by kn-ms — APPROVED** [review]

**Pull request merged**